text/convert.py

```diff
@@ -34,3 +37,125 @@ def run(self):
             for seq_tag, entry in d.items():
                 assert isinstance(entry, str), f"expected str, got {entry!r} (type {type(entry).__name__})"
                 out.write(entry + "\n")
+
+
+class TextDictToStmJob(Job):
+    """
+    Similar as :class:`CorpusToStmJob`, but does not use the Bliss XML, but instead the text dict as input
+    (e.g. via :class:`CorpusToTextDictJob`).
+    """
+
+    def __init__(
+        self,
+        text_dict: Path,
+        *,
+        exclude_non_speech: bool = True,
+        non_speech_tokens: Optional[List[str]] = None,
+        remove_punctuation: bool = True,
+        punctuation_tokens: Optional[Union[str, List[str]]] = None,
```

Comment:
we could even rename it `XXX_tokens_to_remove` to make this more clear.

Reply:
I don't know. I did not think too much about it. If the options don't make sense, why do we have them in `CorpusToStmJob`?

I thought it would be good to have the options also consistent to `CorpusToStmJob` if possible, so it's simple to switch.

Reply:
Ok I also search for `exclude_non_speech` and `remove_punctuation` in i6_experiments and I also never see that they are set. So yea, maybe we can just remove them.

I see `non_speech_tokens` (and maybe also `punctuation_tokens`) being used a few times. So the question would be whether we change the name now and break consistency, or not.

Reply:
I simplified that now. I removed `exclude_non_speech` and `remove_punctuation` and renamed the others to `remove_XXX`.